scribble-lib/scribble/acmart.rkt

```diff
@@ -159,6 +160,20 @@
    abstract-style
    (decode-flow strs)))
 
+(define (extract-abstract p)
+  (unless (part? p)
+    (error 'include-abstract "doc binding is not a part: ~e" p))
+  (unless (null? (part-parts p))
+    (error 'include-abstract "abstract part has sub-parts: ~e" (part-parts p)))
+  (when (part-title-content p)
+    (error 'include-abstract "abstract part has title content: ~e" (part-title-content p)))
+  (part-blocks p))
+
+(define-syntax-rule (include-abstract mp)
+  (begin
+    (require (only-in mp [doc abstract-doc]))
+    (make-nested-flow abstract-style (extract-abstract abstract-doc))))
```

Comment:
Can this call the `abstract`function?

Reply:
Sadly no. Because we don't want it to re-sanitize the string. We 'could' add a secret extra parameter to do that, but...meh?